conformance/README.md

```diff
@@ -5,6 +5,7 @@
 * [kind](https://kind.sigs.k8s.io/).
 * Docker.
 * Golang.
+* [yq](https://github.com/mikefarah/yq/#macos--linux-via-homebrew)
```

Comment:
Can you also add this as a prereq in the developer quickstart guide? https://github.com/nginxinc/nginx-kubernetes-gateway/blob/main/docs/developer/quickstart.md?

Now that I'm looking at that guide, it would also be nice to have a section on running the conformance tests. Do you mind adding that?

Reply:
We have a link to the README in the testing guide - do we want to duplicate the instructions in the guide?

Reply:
I think it's worthwhile to have a section in the quickstart that points to this readme. The quickstart is meant to be a guide to get a development env running and running the conformance tests is an important part of that.

Reply:
Sorry, I misread your first comment 🤦🏼‍♀️ Yep, can do!